src/Core/AdminConsole/Entities/Organization.cs

```diff
@@ -268,5 +268,6 @@ public void UpdateFromLicense(OrganizationLicense license, bool flexibleCollecti
         SmSeats = license.SmSeats;
         SmServiceAccounts = license.SmServiceAccounts;
         LimitCollectionCreationDeletion = !flexibleCollectionsIsEnabled || license.LimitCollectionCreationDeletion;
+        AllowAdminAccessToAllCollectionItems = !flexibleCollectionsIsEnabled || license.AllowAdminAccessToAllCollectionItems;
```

Comment:
The naming here isn't very clear, sorry. `flexibleCollectionsIsEnabled` refers to the MVP feature flag. But this is a v1 feature. So you'll need to pass in both:

```cs
public void UpdateFromLicense(OrganizationLicense license, bool flexibleCollectionsMvpIsEnabled, bool flexibleCollectionsV1IsEnabled)
{
   // snip
       LimitCollectionCreationDeletion = !flexibleCollectionsMvpIsEnabled || license.LimitCollectionCreationDeletion;
       AllowAdminAccessToAllCollectionItems = !flexibleCollectionsV1IsEnabled || license.AllowAdminAccessToAllCollectionItems;
```

It's awkward but we'll clean it up after release.